spec-0010/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "SPEC 10 — Changelog"
+number: 10
+date: 2024-06-03
+author:
+  - "Inessa Pawson <inessapawson@gmail.com>"
+discussion: https://discuss.scientific-python.org/t/
+endorsed-by:
+---
+
+## Description
+
+SPEC 10 provides guidelines and best practices for maintaining a changelog file
+for libraries in the Scientific Python ecosystem.
+
+### Core Project Endorsement
+
+<!--
+Briefly discuss what it means for a core project to endorse this SPEC.
+-->
+
+### Ecosystem Adoption
+
+The endorsement of this SPEC signifies your project's support for the guidelines 
+laid out in the document.
+
+## Implementation
+
+Keeping a clear, organized, and human-readable record of notable changes for each 
+version of a Python library is essential for the project’s maintenance and sustainability. 
```

Comment:
Maybe give a few examples here of the circumstances under which changelogs are useful (seeing API changes, new features, bugs fixed, security concerns; it also gives credit to those who contributed to a release, via code or reviews).

Reply:
I think change log philosophy probably also applies to non-Python libraries?

```suggestion
version of a library is essential for the project’s maintenance and sustainability. 
```

Reply:
@pllim Agree. The reason I mentioned "Python" is because this document is intended for the Scientific Python Ecosystem. Wouldn't "package" be a better word if we want to describe it more generally?